src/commands/math/commands.js

```diff
@@ -369,11 +373,11 @@ var SummationNotation = P(MathCommand, function(_, super_) {
   };
 });
 
-LatexCmds['∑'] =
+LatexCmds['?'] =
 LatexCmds.sum =
 LatexCmds.summation = bind(SummationNotation,'\\sum ','&sum;');
 
-LatexCmds['∏'] =
+LatexCmds['?'] =
```

Comment:
> The symbols are there, but because they’re UTF-8 maybe your viewer is converting them to ANSI?

I wonder if this is an encoding difference between windows and mac or something? This isn't purely a display issue on github, because the symbol is showing up as changed in the diff, and the old symbol displays correctly for me, but the new one does not.